Packs/QRadar/Classifiers/classifier-QRadar_-_Generic_Incoming_Mapper.json

```diff
@@ -1,1113 +1,1017 @@
 {
```

Comment:
I can't make it happen because it formats this file in some reason showing all lines have changed.
Only change in file was under this section
        "dbot_classification_incident_type_all": {
            "dontMapEventToLabels": false,
            "internalMapping": {
                "dbotMirrorDirection": {
                    "complex": null,
                    "simple": "mirror_direction"
                },
                "dbotMirrorId": {
                    "complex": null,
                    "simple": "id"
                },
                "dbotMirrorInstance": {
                    "complex": null,
                    "simple": "mirror_instance"
                }
            }
        }
        
      Where the "internalMapping" was empty and I added those fields. didn't touch anything else in file
      @Itay4